build/polyfills-generator.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const browserify = require('browserify');
+const derequire = require('derequire');
+const createFile = require('./shared/create-file');
+
+const inputFile = path.join(
+	__dirname,
+	'..',
+	'lib',
+	'core',
+	'polyfills',
+	'index.js'
+);
+const outputFile = path.join(
+	__dirname,
+	'..',
+	'tmp',
+	'core',
+	'polyfills',
+	'index.js'
+);
+
+async function run() {
```

Comment:
This is a clone of `imports-generator` - https://github.com/dequelabs/axe-core/blob/develop/build/imports-generator.js

I see no reason it has to be though. You should make this a common function, that when given a source and destination and the namespace to mount, it creates these files.

Why have two files with the same code one for creating `axe.imports` and one for creating `axe.polyfills`?